src/table/utils/accessibility-utils.ts

```diff
@@ -216,9 +216,14 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (value: string) => {
+export const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
```

Comment:
I think we can give this its own file. It doesn't have anything to do with accessibility and accessibility-utils is already pretty big